tests/functional/list/test_list.py

```diff
@@ -56,7 +42,7 @@ def expect_given_output(self, args, expectations):
                 else:
                     assert got == expected
 
-    def expect_snapshot_output(self, project):
+    def expect_snapshot_output(self, happy_path_project):  # noqa: F811
```

Comment:
In this test we're actually using the `happy_path_project` fixture argument in the function. Is the `# noqa: F811` necessary for another reason?

Reply:
+1, also curious about this

Reply:
I think for whatever reason flake8 was complaining about all of these in my local dev environment.
Let me try again and see what happened here

Reply:
This is the problem with pytest, I don't see a easy way of getting rid of it.
https://stackoverflow.com/questions/50268284/why-im-getting-f811-error-in-this-code